crates/terminal/src/terminal.rs

```diff
@@ -710,11 +709,6 @@ impl Terminal {
                 }
             }
 
-            InternalEvent::Copy => {
```

Comment:
I don't think we should remove this, as we'd like to be cooperative and give the PTY time to write into the Terminal.

That said, I think the better solution would be to...